pr_check.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -ex
+
+# test jjb index
+./generate-index.py > test.yaml
+/var/lib/jenkins/.local/bin/jenkins-jobs test test.yaml
+
+# change tags in dockerfiles
+sed -i '' -e "s#:latest#:${ghprbActualCommit}#g" base/**/Dockerfile
+# change registry url in dockerfiles
+sed -i '' -e "s#FROM prod.#FROM push.#g" base/**/Dockerfile
+
+for target in $(ls -d build_order/* | xargs realpath); do
+  cd $target
+  img_tag=$(pwd | sed -e "s|$(pwd)/||g" )
+  echo "Building $img_tag"
+  docker build -t push.registry.devshift.net/osio-prod/$img_tag:test-${ghprbActualCommit} .
```

Comment:
why not change this to prod.registry, and skip the sed on L12 changing it from prod to push?

Reply:
I was also thinking about that, +1 less sed's is always better